docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+
+<BlockSwitcher>
+<Block name="Styled Components">
+[Styled Components](https://github.com/styled-components/styled-components) allows you to create visual primitives with actual CSS code directly tied to components.
+
+#### Usage
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const StyledView = styled(View)`
+  background-color: black;
+  font-size: 32px;
+`;
+
+//use it like any other amplify-ui component
+<StyledView color="red" className="my-styled-view">
+  This is my Styled View
+</StyledView>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Styled Components will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. These Amplify styling props will take precedence over Styled Component styling.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const StyledView = styled(View)`
+  color: blue;
+`;
+
+<StyledView color="red">Using Styling props</StyledView>;
+```
+
+In the example above, the color of the view will be set to `red` because of the Amplify styling prop. [Here]() is a list of all Amplify styling props.
+
+##### Amplify variation props
+
+These props change the look and/or behavior of certain Amplify components. Examples include `size` and `variation`.
+
+```jsx
+import React from 'react';
+import styled from "styled-components";
+import { Button } from '@aws-amplify/ui-react';
+
+const StyledButton = styled(Button)`
+  color: blue
+`;
+
+<StyledButton variation="primary'>Primary Button</StyledButton>
+```
+
+In the example above, the color of the button will be set to `white`, because the primary variation of the button has a color set to `white` and it takes precedence over the Styled Components styling. A list of Amplify variations can be found on [here]().
+
+##### Amplify Custom ClassNames
+
+These are custom classnames added onto Amplify components and can be used with CSS styling rules to modify the look of an Amplify component.
+
+```jsx
+import React from 'react';
+import styled from 'styled-components';
+import { View } from '@aws-amplify/ui-react';
+
+const styledView = styled(View)`
+  color: blue;
+`;
+
+<styledView className="my-styled-view">My Styled View</styledView>;
+```
+
+```CSS
+/* External Style Sheet */
+.my-styled-view {
+  color: red;
+}
+```
+
+In the example above, the color of the view will be `blue` because the stlyed component will take precedence over the classname CSS.
```

Comment:
```suggestion
In the example above, the color of the view will be `blue` because the styled component will take precedence over the classname CSS.
```

Reply:
Can you add an explanation of why the styled component CSS will take precedence over the className CSS?